pysat/_instrument.py

```diff
@@ -831,25 +1024,53 @@ def load(self, yr=None, doy=None, date=None, fname=None, fid=None,
             #print (first_pad, first_time, last_time, last_pad)
 
             # pad data based upon passed parameter
-            if (not self._prev_data.empty) & (not self.data.empty):
-                padLeft = self._prev_data.loc[first_pad : self.data.index[0]]
-                if len(padLeft) > 0:
-                    if (padLeft.index[-1] == self.data.index[0]) :
-                        padLeft = padLeft.iloc[:-1, :]
-                    self.data = pds.concat([padLeft, self.data])
-
-            if (not self._next_data.empty) & (not self.data.empty):
-                padRight = self._next_data.loc[self.data.index[-1] : last_pad]
-                if len(padRight) > 0:
-                    if (padRight.index[0] == self.data.index[-1]) :
-                        padRight = padRight.iloc[1:, :]
-                    self.data = pds.concat([self.data, padRight])
+            # if (not self._empty(self._prev_data)) & (not self.empty):
+            #     padLeft = self._prev_data.loc[first_pad : self.index[0]]
+            #     if len(padLeft) > 0:
+            #         if (padLeft.index[-1] == self.index[0]) :
+            #             padLeft = padLeft.iloc[:-1, :]
+            #         # self.data = self.concat_data([padLeft, self.data])
+            #     print (padLeft)
```

Comment:
I think we decided previously not to leave old methods as commented out code?  If you do want to keep this bit in, some expaination should be added before the commented out code.

Reply:
You are 100% correct. Removed commented code.